src/test/java/com/eppo/sdk/deserializer/BanditsDeserializerTest.java

```diff
@@ -0,0 +1,127 @@
+package com.eppo.sdk.deserializer;
+
+import com.eppo.sdk.dto.*;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.apache.commons.io.FileUtils;
+import org.junit.jupiter.api.Test;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.Map;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+class BanditsDeserializerTest {
```

Comment:
This test is a doozy because I'm checking every property from the [`bandits-paremeters-1.json` test JSON file](https://github.com/Eppo-exp/java-server-sdk-bandit-beta/pull/3/files#diff-57812398e9d2358377168e236f99ba934e2d20a6a01727673965263e4476b1be).